lib/chef-dk/command/verify.rb

```diff
@@ -75,25 +75,37 @@ def components
       #
       add_component "berkshelf" do |c|
         c.gem_base_dir = "berkshelf"
-        # For berks the real command to run is "bundle exec thor spec:ci"
+        # For berks the real command to run is "#{bin("bundle")} exec thor spec:ci"
         # We can't run it right now since graphviz specs are included in the
         # test suite by default. We will be able to switch to that command when/if
         # Graphviz is added to omnibus.
-        c.unit_test { sh("bundle exec rspec --color --format progress spec/unit --tag ~graphviz") }
-        c.integration_test { sh("bundle exec cucumber --color --format progress --tags ~@no_run --tags ~@spawn --tags ~@graphviz --strict") }
+        c.unit_test do
+          sh("#{bin("bundle")} install")
```

Comment:
Yep. The idea here is, we shouldn't have to ship all the gems that are used just to verify the production components. Basically, you pay a price if you want to `chef verify`--you will have to install the test components. But at least 99% of our users who don't run it, won't :)